frontend/src/app/modules/boards/boards-sidebar/boards-menu.component.ts

```diff
@@ -7,6 +7,7 @@ import {AngularTrackingHelpers} from "core-components/angular/tracking-functions
 import {MainMenuNavigationService} from "core-components/main-menu/main-menu-navigation.service";
 import {map} from "rxjs/operators";
 import {CurrentProjectService} from "core-components/projects/current-project.service";
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 
 export const boardsMenuSelector = 'boards-menu';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/21a3227c049011c1dde88d6b831a80dc9baff9ee/frontend/src/app/modules/boards/boards-sidebar/boards-menu.component.ts#L13"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>